### PR TITLE
replace .Active by .Enabled

### PR DESCRIPTION
**Pull request by vieux**
ping @tiborvass @anusha-ragunathan 

related to  https://github.com/docker/docker/pull/25424

### CODE REVIEW

**Comment by tiborvass** [comment]
*Cross posting from https://github.com/docker/docker/pull/25424#issuecomment-237715136:*
I wonder if it should be State: Enabled instead.

**Comment by anusha-ragunathan** [comment]
LGTM

File: types/plugin.go [hunk 1]
```diff
@@ -29,7 +29,7 @@ type Plugin struct {
 	ID       string `json:"Id,omitempty"`
 	Name     string
 	Tag      string
-	Active   bool
+	Enabled  bool
```
Comment (stevvooe) [inline types/plugin.go]: I know we don't do this in this project, but could we _please_ document our fields? What does enabled actually mean?

**Comment by stevvooe** [comment]
LGTM after documenting the field

I am not convinced a state machine will help here, without developing a proper model around the state machine.

**Comment by vieux** [comment]
PTAL @stevvooe

**Comment by tiborvass** [comment]
LGTM

**Comment by stevvooe** [comment]
LGTM

**Pull request merged**